### DISCUSSION TITLE
Ability to switch between multiple configured clients

### DISCUSSION THREAD

**Discussion by hwillson** [apollographql/apollo-client-devtools#373] [Ideas]
Migrated from: https://github.com/apollographql/apollo-client-devtools/issues/20

I'm experimenting with Apollo in the context of Angular 2 app. I connect to multiple graphql end-points , so I have configured multiple clients using ApolloModule.forRoot(provideClients).
Not sure if this is possible, but it would be nice if devtools shows a drop down showing available clients to use, whatever the current context in the webapp might be.

**Comment by tortilaman** [comment]
I would find this incredibly useful. I'm working with Apollo in an environment with multiple MFEs running using module federation, and multiple Apollo clients. I realize there is probably a way to configure things differently to consolidate things, but that is outside my control.

In my chrome devtools console I see there are three instances of the Apollo devtools extension running, but I only see one Apollo tab, and it's for the client that isn't relevant to my work. I would really appreciate this functionality, I'm happy to contribute and / or help test.
